### DISCUSSION TITLE
Swiss Electricity Production

### DISCUSSION THREAD

**Discussion by Alberto4412** [electricitymaps/electricitymaps-contrib#2817] [Q&A] [answered]
Why is the electricity production (water, nuclear, solar) in Switzerland not shown - just like in other european countries. The data is available at ENTSOE.  Thanks and regards al steiner

**Comment by FelixDQ** [comment, marked as answer]
Unfortunately, ENTSOE gives inaccurate data for Hydro and Hydro Storage We don't want to mislead users, so we have chosen not to color the zone on the map.

See more here: https://github.com/tmrowco/electricitymap-contrib/issues/892

**Comment by ctx2kCH** [comment]
Hi all, I have no idea if the data can be usable or not. But there is a current map with all production facilities in Switzerland. Maybe the question marks about the possible production capacity can be solved. 
It is not live data of the plants, but at least something. 

[https://www.geocat.ch/geonetwork/srv/ger/md.viewer#/full_view/e5a00bdb-5022-4856-ad4a-d1afe7bf38b0/tab/complete](url)
[ch.bfe.elektrizitaetsproduktionsanlagen.zip](https://github.com/electricitymap/electricitymap-contrib/files/8239041/ch.bfe.elektrizitaetsproduktionsanlagen.zip)
